### DISCUSSION TITLE
[Q&A] How to renew certificate's expiration date after freezing project?

### DISCUSSION THREAD

**Discussion by nunziosorrentino** [NVIDIA/NVFlare#2688] [Q&A] [answered]
### Python version (`python3 -V`)

3.8

### NVFlare version (`python3 -m pip list | grep "nvflare"`)

2.2.5

### NVFlare branch (if running examples, please use the branch that corresponds to the NVFlare version, `git branch`)

2.2

### Operating system

Ubuntu 18.04

### Have you successfully run any of the following examples?

- [ ] hello-numpy-sag with simulator
- [X] hello-pt with simulator
- [ ] hello-numpy-sag with POC
- [ ] hello-pt with POC

### Please describe your question

One year ago I created a nvflare dashboard with many clients. Now I discovered that the certificates expire after one year and I can't run the federation with new clients that were ongoing added. Is it possible to update the expiration date without forcing clients to re-register on a new dashboard?

**Comment by YuanTingHsieh** [comment]
@IsaacYangSLA can you provide more information on this one?
How can user do a new provision using dashboard?

Thanks!

**Comment by YuanTingHsieh** [comment, marked as answer]
@nunziosorrentino thanks for sharing your experience!

Unfortunately for the version of 2.2.5, you will need to re-register on a new dashboard, basically setup the whole FL system again.

We will think how to enhance this in later version.
Another thing, if you got chance, you can try our newer release and with new features, and let us know how it performs.

**Reply by nunziosorrentino** [reply]
Dear @YuanTingHsieh thank you for the answer.

I am actively working on a new version of our federated learning project with the latest version of Nvidia Flare. This new package is still being developed and tested using the "nvflare simulation" tool. So for the real federation with clients, I still use the original package that uses nvflare 2.2, so I will create a new dashboard and re-register all clients. 

I will let you know once the new one is ready for real federation.
